recipes/libcurl/all/conanfile.py

```diff
@@ -112,7 +112,10 @@ def system_requirements(self):
         pass
 
     def build_requirements(self):
-        if self._is_win_x_android:
+        if self._is_mingw and "CONAN_BASH_PATH" not in os.environ and \
+           tools.os_info.detect_windows_subsystem() != "msys2":
+            self.build_requires("msys2/20190524")
+        elif self._is_win_x_android:
             self.build_requires("ninja/1.9.0")
```

Comment:
not sure why ninja is hard requirement for Android, but it appears to be irrelevant

Reply:
https://github.com/conan-io/conan-center-index/pull/1585#discussion_r426700559

Reply:
that doesn't explain why it needs to use Ninja generator for Android, I believe it's okay to use Make as well, but since it's offtopic, we can discuss in another PR